wandb/cli/cli.py

```diff
@@ -650,7 +650,31 @@ def _clean():
 @click.option("--program", default=False, help="Set sweep program")
 @click.option("--settings", default=False, help="Set sweep settings", hidden=True)
 @click.option("--update", default=None, help="Update pending sweep")
-@click.argument("config_yaml")
+@click.option(
+    "--stop",
+    is_flag=True,
+    default=False,
+    help="Finish a sweep to stop running new runs and let currently running runs finish.",
+)
+@click.option(
+    "--cancel",
+    is_flag=True,
+    default=False,
+    help="Cancel a sweep to kill all running runs and stop running new runs.",
+)
+@click.option(
+    "--pause",
+    is_flag=True,
+    default=False,
+    help="Pause a sweep to temporarily stop running new runs.",
+)
+@click.option(
+    "--resume",
+    is_flag=True,
+    default=False,
+    help="Resume a sweep to continue running new runs.",
```

Comment:
```suggestion
    help="Resume a paused or stopped sweep: continue running new runs.",
```

Reply:
Stopped sweeps can be resumed?

Reply:
Yeah I think only paused can, nice catch. 

Resume a paused sweep. This command instructs a running sweep agent process to resume requesting new work. This command can be undone with "pause."